.yarnrc.yml

```diff
@@ -4,7 +4,7 @@ enableGlobalCache: false
 
 enableScripts: false
 
-enableTelemetry: 0
+enableTelemetry: false
```

Comment:
Hmm this aligns with core but it seems the docs say it should be `0` https://yarnpkg.com/advanced/telemetry. Revert this change it's for improving `yarn`. I'm actually not opposed to this being turned on 🤷

Reply:
Updated https://github.com/MetaMask/metamask-design-system/pull/17/commits/991fc6ea41712891f00f4181f50eb8dc4c224ad7